.circleci/config.yml

```diff
@@ -68,6 +68,15 @@ jobs:
       - run:
           name: Create local DynamoDB referrals table
           command: aws dynamodb create-table --cli-input-json file://./config/tables/referrals.json --endpoint-url http://localhost:8000 > /dev/null
+      - run:
+          name: Add data to DynamoDB referrals table
+          command: |
+            aws dynamodb put-item --table-name referrals --item file://./config/tables/content/sent.json --endpoint-url http://localhost:8000 > /dev/null
+            aws dynamodb put-item --table-name referrals --item file://./config/tables/content/rejectedWithComments.json --endpoint-url http://localhost:8000 > /dev/null
+            aws dynamodb put-item --table-name referrals --item file://./config/tables/content/rejected.json --endpoint-url http://localhost:8000 > /dev/null
+            aws dynamodb put-item --table-name referrals --item file://./config/tables/content/accepted.json --endpoint-url http://localhost:8000 > /dev/null
+            aws dynamodb put-item --table-name referrals --item file://./config/tables/content/toBeAccepted.json --endpoint-url http://localhost:8000 > /dev/null
+            aws dynamodb put-item --table-name referrals --item file://./config/tables/content/toBeRejected.json --endpoint-url http://localhost:8000 > /dev/null
```

Comment:
This is starting to become huge. Maybe we could extract that into some shell script on the project root & then run that?

Reply:
Thinking it over... it could be something for refactoring session that we've scheduled.

Reply:
I'm looking at reducing the number of files right now by reseting their values after the tests run, so it should be a few less lines, but if that's still too much, can look at putting it somewhere separate

Reply:
I think we can ignore nitpick this for now, as it seems it's causing issues with the tests.